include/Common/Platform/POSIX/POSIXError.h

```diff
@@ -18,8 +18,9 @@
 #pragma once
 #include <optional>
 #include <string_view>
+#include <cerrno>
```

Comment:
Why include <cerrno> if you changed it to `int`?

Reply:
Errno is provided by cerrno, but `errno_t` is nonstandard